modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -1131,6 +1135,30 @@ void VisualScriptBuiltinFunc::exec_func(BuiltinFunc p_func, const Variant **p_in
 
 			*r_return = String(result);
 
+		} break;
+		case VisualScriptBuiltinFunc::TEXT_ORD: {
+
+			if (p_inputs[0]->get_type() != Variant::STRING) {
+				r_error.error = Variant::CallError::CALL_ERROR_INVALID_ARGUMENT;
+				r_error.argument = 0;
+				r_error.expected = Variant::STRING;
+				
```

Comment:
Travis is complaining about trailing tabs here, could you verify if there's anything here in your end?

Reply:
I noticed now that there are trailing tabs there on line 1145. I will push the change along with moving `TEXT_ORD` to the bottom of the enum as requested by @bojidar-bg . That should hopefully fix all the issues.